Sources/PointFree/Routers/PointFreeRoutes.swift

```diff
@@ -21,6 +21,7 @@ public enum Route: DerivePartialIsos {
   case logout
   case paymentInfo
   case pricing(String?, Int?)
+  case reactivate
```

Comment:
Added a `reactivate` endpoint for subscriptions that are canceling this period but haven't canceled yet.